pkg/actuators/machine/machine.go

```diff
@@ -302,7 +302,17 @@ func (ms *machineScope) delete() error {
 		}
 		return errors.Wrap(err, "error finding VM by name")
 	}
-	return ms.ovirtClient.RemoveVM(vm.ID(), ovirtC.ContextStrategy(ms.Context))
+	if err := vm.Stop(true, ovirtC.ContextStrategy(ms.Context)); err != nil {
```

Comment:
What happens here if the VM is already stopped? If it would return an error, we should handle that.

Reply:
we have tested this scenario, this won't trigger any errors.